concepticondata/conceptlists.tsv

```diff
@@ -239,3 +239,4 @@ Carling-2017-97j	Carling, Gerd and Larsson, Filip and Cronhamn, Sandra and Farre
 Carling-2017-87k	Carling, Gerd and Larsson, Filip and Cronhamn, Sandra and Farren, Rob and Aliyev, Elnur and Johansson, Niklas and Chukhua, Merab and Goergens, Anne and Vamling, Karina and Cathcart, Chundra and Tetraze, Maka and Lomadze, Tamuna and Holmer, Arthur and Avidzba, Leila and Gvantseladze, Teimuraz and Khalilov, Madzhid and Cabral, Ana Suelly Arruda Câmara and Mana, Joaquim and Sinha, Vera da Silva and Sampaio, Wany and Sinha, Chris and Gakran, Nanblá and Sabino, Wary Kamaiurá and Kamaiura, Aisanain Páltu and Nahukua, Kaman and Mehinako, Makaulaka	2017	k	87	basic,areal	English	Turkic languages	https://diacl.ht.lu.se/WordList/Index	Carling2017		A list of basic cultural terms for the Turkic languages.		
 Nagano-2013-1256	Nagano, Yasuhiko and Prins, Marielle	2013		1256	questionnaire,areal	English	South-East Asian Languages	https://htq.minpaku.ac.jp/databases/rGyalrong/lang/area/index.php?langindex=eng	Nagano2013		The results of an international joint survey of the rGyalrongic languages, originally published by the authors in their own website.		
 Yang-2011-1014	Yang, Cathryn	2011		1014	questionnaire	Chinese	Lalo Languages	https://www.sil.org/system/files/reapdata/45/63/47/45634767234331504727329755903316120678/Yang_Lalo_Sept_2011.pdf	Yang2011		A survey of Lalo regional varieties for research in phylogenetics, dialectometry and sociolinguistics.		
+Vejdemo-2016-167	Vejdemo, Susanne and Hörberg, Thomas	2016		167	basic	English	Global	https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147924#pone.0147924.ref032 	Vejdemo2016		This list represents 167 content words, built upon the list by [Pagel et al. (2007)](:ref:Pagel-2007-200). It was originally compiled with the purpose of assessing the rate of lexical replacement for the specified concepts in English, Danish, Dutch, German and Swedish, according to word class, frequency, age of acquisition, synonyms, arousal, imageability and average mutual information either from published databases or from corpora and lexica.		
```

Comment:
As this is not only a basic list, but also a list with numbers, please add "acquisition" as tag, and compare with similar lists of this types, maybe even adding a new tag "speech norms". @chrzyki, please could you see that you help, as you added a similar list with speech norms before?

Reply:
@LinguList I have just looked at the lists with the acquisition tag on Concepticon and they are all based on the studies specifically focused on the acquisition of the listed concepts by children. In contrast, this particular study by Vejdemo investigates the likelihood of lexical replacement ("borrowability") of the concepts, based on a range of criteria like frequency, imageability, arousal and the age of acquisition, among them. So technically, it's an analysis of the stability of these concepts in a number of languages. The ranked tag also seems appropriate since Pagel's list which is the basis for Vejdemo's list is also tagged as ranked